modules/df/df_tools/df_tools_translation/df_tools_translation.info.yml

```diff
@@ -4,7 +4,4 @@ description: Contains configuration neccessary to enable entity translation.
 core_version_requirement: ^9 || ^10
 package: Demo Framework Tools
 dependencies:
-  - config_translation
```

Comment:
@vaibhav-acquia why are we removing dependencies?

Reply:
@kevinfunk As we have discussed before that these are deprecated dependencies and not part of composer.json and composer.lock file as well..do you see any potential implications of removing it?

PS: https://www.drupal.org/project/config_translation

Reply:
Starting with Drupal 8, they are part of Drupal core https://git.drupalcode.org/project/drupal/-/tree/11.x/core/modules/config_translation?ref_type=heads and https://git.drupalcode.org/project/drupal/-/tree/11.x/core/modules/content_translation?ref_type=heads. We are using both modules on all profiles that have translations.